Public/js/search.js

```diff
@@ -299,6 +299,44 @@ function autocompleteStepArtistSearch(stepperFields, packJson) {
 	return res; // Return the array of results that match all the conditions
 }
 
+/**
+ * File Status function that allows searching by song status.
+ * @parameter statusName name of the status requested, can be "Rejected", "Conditional Queue", "Accepted", and "Released"
+ * @parameter packJson The JSON of song packs as keys, with array of song objects for each key.
+ * @return result as an JSON array of songs that matched the input for status.
+ * @return -1 input no files with requested status
+ */
+function getFilesByStatus(statusName, packJson) {
+
+    //initialize the packs variable, and return array.
+	const packs = packJson;
+	const res = [];
```

Comment:
I would rename all of these return values to be "results" but that's just preference